cli.go

```diff
@@ -368,20 +358,32 @@ func printResult(out io.Writer, result *Result, mode DisplayMode, withStats bool
 	}
 }
 
-// buildDdlStatements build batched statement only if all statements are DDL statements
-func buildDdlStatements(input string) Statement {
-	var ddls []string
+func buildCommands(input string) ([]*command, error) {
+	var stmts []*command
```

Comment:
nit: `cmds` instead of `stmts`?

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/40/commits/99a58f3fa9aeb74933b0d743a8e68772f000aeee